frontend/src/App.tsx

```diff
@@ -185,6 +185,30 @@ class App extends PureComponent<Props, State> {
     MetricsManager.current.enqueue("viewReport")
   }
 
+  showError(title: string, errorNode: ReactNode): void {
+    logError(errorNode)
+    const newDialog: DialogProps = {
+      type: DialogType.WARNING,
+      title: title,
+      msg: errorNode,
+      onClose: () => {},
+    }
+    this.openDialog(newDialog)
+  }
+
+  isServerVersionUpdated(initializeMsg: Initialize): boolean {
```

Comment:
Please, canyou add docstring to this func and maybe check the name? It's hard to understand :)